include/mbedtls/ecp.h

```diff
@@ -62,6 +62,7 @@ typedef enum
     MBEDTLS_ECP_DP_BP384R1,        /*!< 384-bits Brainpool curve */
     MBEDTLS_ECP_DP_BP512R1,        /*!< 512-bits Brainpool curve */
     MBEDTLS_ECP_DP_CURVE25519,           /*!< Curve25519               */
+    MBEDTLS_ECP_DP_CURVE448,             /*!< Curve448 (Ed448)         */
```

Comment:
@NWilson We only support ECDHE-Curve448, but not Ed448 == EdDSA-Curve448, so I'd prefer the comment to not mention `Ed448`.